ecs-logs-collector.sh

```diff
@@ -128,35 +127,64 @@ is_root()
   ok
 }
 
-is_diskfull()
-{
+is_diskfull() {
   try "check disk space usage"
 
   threshold=70
   i=2
   result=`df -kh |grep -v "Filesystem" | awk '{ print $5 }' | sed 's/%//g'`
```

Comment:
IMO we should probably not try to parse this output at all, but should just include the raw output in the report. Probably out of scope for this PR.